modules/ROOT/pages/publish-api-project-to-exchange.adoc

```diff
@@ -0,0 +1,19 @@
+= Publish an API Project to Exchange
+
+You can publish your API project as a new version of an API specification to Exchange.
+
+To publish your API project as an API in Exchange:
+
+. In the Package Explorer view, right-click your API specification project and select *Manage API Specification* > *Publish to Exchange*.
```

Comment:
@fermujica I discovered this in the style guide last night...I guess our style is to do **Manage API Specification > Publish to Exchange**, essentially highlighting the directional arrow as well. I've always done it the way you do...